src/main/java/de/tum/cit/aet/artemis/buildagent/BuildAgentConfiguration.java

```diff
@@ -55,6 +63,24 @@ public BuildAgentConfiguration(ProgrammingLanguageConfiguration programmingLangu
         this.programmingLanguageConfiguration = programmingLanguageConfiguration;
     }
 
+    @EventListener(ApplicationReadyEvent.class)
```

Comment:
should we move this code to the constructor instead? in case another service uses them right after the application starts.

Reply:
Can't be done since some values aren't present during the construction